src/main/scala/com/github/music/of/the/ainur/almaren/neo4j/Neo4jConnector.scala

```diff
@@ -0,0 +1,40 @@
+package com.github.music.of.the.ainur.almaren.neo4j
+
+import org.apache.spark.sql.{DataFrame,SaveMode}
+import com.github.music.of.the.ainur.almaren.Tree
+import com.github.music.of.the.ainur.almaren.builder.Core
+import com.github.music.of.the.ainur.almaren.state.core.{Target,Source}
+
+private[almaren] case class SourceNeo4j(url: String, options:Map[String,String] = Map()) extends Source {
+  def source(df: DataFrame): DataFrame = {
+    logger.info(s"url:{$url}")
+    df.sparkSession.read.format("org.neo4j.spark.DataSource")
```

Comment:
Let's add username and password like we have for JDBC, the difference is that we don't log the password. Look sourceJdbc and you will undertand.

Reply:
Requested changes are implemented